spec/asyncapi.md

```diff
@@ -918,6 +939,18 @@ bindings:
 
 
 
+#### <a name="operationReplyObject"></a>Reply Object
+
+Describes the reply part that MAY be applied to an Operation Object. If an operation implements the request/reply pattern, the reply object represents the response message.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="operationReplyObjectAddress"></a>address | [Correlation ID Object](#correlationIdObject) &#124; [Reference Object](#referenceObject) | Definition of the reply address used telling operation where to send the response to. Used if a client uses a inbox for request/reply responses.
```

Comment:
As we aggreged i use here the correlation ID Object.
or do you vote for:
- duplicate the "Correlation ID Object" to " Reply Address Object" but with same structure?
- find a better name for "Correlation ID Object"

Reply:
@derberg Please comment

Reply:
I would just duplicate it even if it has the same structure. We never know how they will evolve over time and having duplicate things in the spec is not really a big issue as it might be in code.

Reply:
done